package.json

```diff
@@ -149,7 +149,7 @@
     "uglifyjs-webpack-plugin": "^1.0.0",
     "unused-files-webpack-plugin": "^3.0.0",
     "webpack": "^5.37.0",
-    "webpack-dev-server": "^2.9.1",
+    "webpack-dev-server": "^3.11.2",
```

Comment:
# Upgrade `webpack-dev-server` to the latest version

`yarn upgrade webpack-dev-server@latest`
```
warning webpack-dev-server > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
[3/5] 🚚  Fetching packages...
[4/5] 🔗  Linking dependencies...
warning " > @cypress/webpack-preprocessor@4.1.5" has incorrect peer dependency "webpack@^4.18.1".
warning " > @testing-library/cypress@5.3.1" has incorrect peer dependency "cypress@^2.1.0 || ^3.0.0 || ^4.0.0".
warning " > babel-loader@7.1.5" has incorrect peer dependency "webpack@2 || 3 || 4".
warning " > extract-text-webpack-plugin@3.0.2" has incorrect peer dependency "webpack@^3.1.0".
warning " > html-webpack-plugin@2.30.1" has incorrect peer dependency "webpack@1 || ^2 || ^2.1.0-beta || ^2.2.0-rc || ^3".
warning " > postcss-cssnext@2.11.0" has unmet peer dependency "caniuse-db@^1.0.30000652".
warning " > uglifyjs-webpack-plugin@1.3.0" has incorrect peer dependency "webpack@^2.0.0 || ^3.0.0 || ^4.0.0".
```

Running `yarn-dev-ee` after this produces the following error:
```
[frontend] CLI for webpack must be installed.
[frontend]   webpack-cli (https://github.com/webpack/webpack-cli)
```